doc/system-telemetry/reboot-cause.md

```diff
@@ -0,0 +1,150 @@
+# Reboot-cause information via telemetry agent
+
+## Revision
+
+| Rev | Date     | Author      | Change Description |
+|:---:|:--------:|:-----------:|--------------------|
+| 0.1 | 09/02/20 | Sujin Kang  | Initial version    |
+
+## Scope
+-Enable sonic streaming telemetry agent to send Reboot-cause information
```

Comment:
The document describes how we will put reboot-cause information into State DB, but doesn't mention the telemetry agent. I think you should at least mention here that this document describes how we will store reboot-cause information in State DB so that it can be retrieved by the streaming telemetry agent.